R/pkg/R/utils.R

```diff
@@ -385,6 +385,41 @@ getStorageLevel <- function(newLevel = c("DISK_ONLY",
                          "OFF_HEAP" = callJStatic(storageLevelClass, "OFF_HEAP"))
 }
 
+storageLevelToString <- function(levelObj) {
+  useDisk <- callJMethod(levelObj, "useDisk")
+  useMemory <- callJMethod(levelObj, "useMemory")
+  useOffHeap <- callJMethod(levelObj, "useOffHeap")
+  deserialized <- callJMethod(levelObj, "deserialized")
+  replication <- callJMethod(levelObj, "replication")
+  if (!useDisk && !useMemory && !useOffHeap && !deserialized && replication == 1) {
```

Comment:
python has itself `StorageLevel` class, and the python side code about `storageLevel`  also exists  duplicated code problem...
and if we make an r-side `StorageLevel` class may cause the code more complex and seems won't help solving the duplicated code problem.
What do you think about it ?